DEVELOPMENT.md

```diff
@@ -0,0 +1,11 @@
+# How to Develop Proxy Wasm Cpp Host
+
+## Generate compile commands
+
+You could use the following commands to generate the `compile_commands.json` file:
+
+```
+BAZEL_BUILD_OPTION_LIST="--define=engine=multi" ./tools/gen_compdb.py --include_all //test/... //:lib
+```
+
+Then you could use clangd (or other tools the could parse `compile_commands.json`) to setup editors with cross reference, code completion.
```

Comment:
Suggest moving this to be first, and to include a link and a little more information giving context on why a compilation database is useful. Proposed text:

```
[JSON Compilation Database](https://clang.llvm.org/docs/JSONCompilationDatabase.html) files can be used by [clangd](https://clangd.llvm.org/) or similar tools to add source code cross-references and code completion functionality to editors.

The following command can be used to generate the `compile_commands.json` file:
[...]
```

Reply:
Great.